includes/connect/class-wc-stripe-connect.php

```diff
@@ -265,5 +302,83 @@ private function record_account_connect_track_event( bool $had_error ) {
 			// a queue wouldn't be processed due to the redirect that comes after.
 			WC_Tracks::record_event( $event_name, [ 'is_test_mode' => $is_test ] );
 		}
+
+		/**
+		 * Schedules the App OAuth connection refresh.
+		 *
+		 * @since 8.6.0
+		 */
+		private function schedule_connection_refresh() {
+			if ( ! $this->is_connected_via_app_oauth() ) {
+				return;
+			}
+
+			/**
+			 * Filters the frequency with which the App OAuth connection should be refreshed.
+			 * Access tokens expire in 1 hour, and there seem to be no way to customize that from the Stripe Dashboard:
+			 * https://docs.stripe.com/stripe-apps/api-authentication/oauth#refresh-access-token
+			 * We schedule the connection refresh every 55 minutues.
+			 *
+			 * @param int $interval refresh interval
+			 *
+			 * @since 8.6.0
+			 */
+			$interval = apply_filters( 'wc_stripe_connection_refresh_interval', HOUR_IN_SECONDS - 5 * MINUTE_IN_SECONDS );
+
+			// Make sure that all refresh actions are cancelled before scheduling it.
+			$this->unschedule_connection_refresh();
+
+			as_schedule_single_action( time() + $interval, 'wc_stripe_refresh_connection', [], WC_Stripe_Action_Scheduler_Service::GROUP_ID, true, 0 );
```

Comment:
```suggestion
			as_schedule_single_action( time() + $interval, 'wc_stripe_refresh_connection', [], WC_Stripe_Action_Scheduler_Service::GROUP_ID, false, 0 );
```

This has to be `false` unfortunately. Because we're currently in the scheduled job the action's status is 'processing' or something similar and setting this to `true` will cause this action to not be scheduled while reconnecting. 

eg connect to an account via the app, run the scheduled action and you'll notice it isn't rescheduled.